src/utils/combineReducers.js

```diff
@@ -97,9 +98,16 @@ export default function combineReducers(reducers) {
     }
   });
 
+  if (sanityError) {
+    console.error(sanityError.message);
+  }
+
```

Comment:
I don't think `console.error` is inherently bad if it is a sign of an error. React also uses `console.error` for `propTypes` warnings because they're not really warnings: if you have them, you need to fix them. The nice thing about `console.error` is that it prints the stack trace, unlike its friends.